pywincffi/core/typesbase.py

```diff
@@ -0,0 +1,58 @@
+"""
+Types Base
+==========
+
+Provides the base types on top of which user visible types will be built.
+"""
+
+import cffi
+
+
+# pylint: disable=too-few-public-methods
+class CFFICDataWrapper(object):
+
+    """
+    Base class for exposing Python types and interfaces to pywincffi users.
```

Comment:
nit: newline before the bullet list starts, probably would be nice to have : at the end of this line too.